### PR TITLE
GUI followup for #8148

### PR DESCRIPTION
**Pull request by illusion0001**
Followup for #8148 
Under advanced tab with descrption `Forces the floating point NaN (Not A Number) value outputted from PPU vector instructions, to be accurate to the real hardware. (0x7FC00000)` 
Thanks @VelocityRa 👍

### CODE REVIEW

File: rpcs3/rpcs3qt/tooltips.h [hunk 1]
```diff
@@ -79,6 +79,7 @@ class Tooltips : public QObject
 		const QString accurate_getllar             = tr("Never use this.");
 		const QString accurate_putlluc             = tr("Never use this.");
 		const QString accurate_llvm_dfma           = tr("Provides extra accuracy on FMA instructions at the cost of performance.\nWhile disabling it might give a decent performance boost if your CPU doesn't support FMA, it may also introduce subtle bugs that otherwise do not occur.\nYou can't disable it if your CPU supports FMA.");
+		const QString accurate_vector_nan          = tr("Forces the floating point NaN (Not A Number) values outputted from PPU vector instructions, to be accurate to the real hardware. (0x7FC00000)");
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/tooltips.h:82]:
```suggestion
		const QString accurate_vector_nan          = tr("Forces the floating point NaN (Not A Number) values outputted from PPU vector instructions to be accurate to the real hardware. (0x7FC00000)");
```

File: rpcs3/rpcs3qt/emu_settings_type.h [hunk 2]
```diff
@@ -23,6 +23,7 @@ enum class emu_settings_type
 	AccurateGETLLAR,
 	AccuratePUTLLUC,
 	AccurateLLVMdfma,
+	accurateVectorNaN,
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/emu_settings_type.h:26]:
```suggestion
	AccurateVectorNaN,
```

Comment (illusion0001) [reply rpcs3/rpcs3qt/emu_settings_type.h]: Resolved

**Comment by Megamouse** [comment]
you forgot to change the usages of the enum value

**Comment by illusion0001** [comment]
fixed

**Pull request merged**